book/src/error-msg.md

```diff
@@ -57,3 +81,5 @@ to the users.
 [use-error]: ./use-error.html
 [err-msg-api]: https://boats.gitlab.io/failure/doc/failure/fn.err_msg.html
 [format-err-api]: https://boats.gitlab.io/failure/doc/failure/macro.format_err.html
+[bail-api]: https://boats.gitlab.io/failure/doc/failure/macro.bail.html
+[ensure-api]: https://boats.gitlab.io/failure/doc/failure/macro.bail.html
```

Comment:
You've linked `macro.bail.html` twice here, I think you meant this one to be `macro.ensure.html`.

Reply:
fixed